### ISSUE TITLE
Minor issue with example in docs

### ISSUE THREAD

**Issue by danabrey** [Happyr/Doctrine-Specification#165]
In this example (https://github.com/Happyr/Doctrine-Specification#querybuilder), I think it's missing the re-defining of the `$qb` variable as the result of the methods.

```php
    public function myQuery(Company $company)
    {
        $qb = $this->em->getRepository('HappyrRecruitmentBundle:Advert')->createQueryBuilder('r');
        $this->filterAdvertsWeShouldClose($qb)
        $this->filterOwnedByCompany($qb, $company)

        return $qb->getQuery()->getResult();
    }
```

should be

```php
    public function myQuery(Company $company)
    {
        $qb = $this->em->getRepository('HappyrRecruitmentBundle:Advert')->createQueryBuilder('r');
        $qb = $this->filterAdvertsWeShouldClose($qb)
        $qb = $this->filterOwnedByCompany($qb, $company)

        return $qb->getQuery()->getResult();
    }
```

**Comment by Nyholm** [comment]
I understand that the docs are confusing. You dont actually need a return statement of `filterAdvertsWeShouldClose`. That should be removed.

**Comment by peter-gribanov** [comment]
Fixed in #167
